lib/internal/Magento/Framework/Cache/Config/Reader.php

```diff
@@ -45,4 +50,25 @@ public function __construct(
             $defaultScope
         );
     }
+
+    /**
+     * Load configuration scope
+     *
+     * @param string|null $scope
+     * @return array
+     * @throws \Magento\Framework\Exception\LocalizedException
+     */
+    public function read($scope = null)
+    {
+        try {
+            return parent::read($scope);
+        } catch (\Magento\Framework\Exception\LocalizedException $e) {
```

Comment:
May I ask you why do we need to transform this localized exception and throw once again?

Thank you

Reply:
Hello @rogyar.
Thank you for your remark.
I have added the render-method that prepares an error message to output error messages correctly.
Please recheck.
Thank you in advance.